### DISCUSSION TITLE
[FEATURE] DressNPC on AC

### DISCUSSION THREAD

**Discussion by ghost** [azerothcore/azerothcore-wotlk#3886] [Ideas]
I propose to add DressNPC natively to AzerothCore, the system that Rochet2 has developed is really handy for small servers that want to have custom pnj's very easily.

Moreover, it doesn't overwrite the existing display system so the CreatureDisplayInfo table is not obsolete.
These scripts could be added natively like the MultiVendor.

I have already implemented DressNPC on the core of my private server, if you want to test this feature: 
https://github.com/WarlogServer/Core/commit/a91f319f7985e7e980c737786481c24af09518d1

I think this is where it belongs at AzerothCore.

**Comment by FrancescoBorzi** [comment]
it would be nice if you can make a Module out of it. If you need new hooks inside the core to support it, you can merge them.

**Comment by ghost** [comment]
I have absolutely no idea how to make a module out of it ^^'
